include/wlr/render.h

```diff
@@ -5,13 +5,17 @@
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 #include <wayland-server-protocol.h>
+#include <wlr/types/wlr_box.h>
 #include <wlr/types/wlr_output.h>
 
 struct wlr_texture;
 struct wlr_renderer;
 
 void wlr_renderer_begin(struct wlr_renderer *r, struct wlr_output *output);
 void wlr_renderer_end(struct wlr_renderer *r);
+void wlr_renderer_clear(struct wlr_renderer *r, float red, float green,
+	float blue, float alpha);
```

Comment:
Depending on the backend, this could make sense. For instance on the Wayland backend one could configure its desktop with a transparent background color (Wayland guys are also talking about transparent physical OLED screens).

Reply:
Maybe that should take a `[4]float` instead to be consistent with other functions.